src/gui/maindialog.h

```diff
@@ -22,17 +33,20 @@ class MyFrame: public wxFrame {
 	static void onLog(const char *);
 
 private:
+	void onSysTrayClick(wxTaskBarIconEvent &event);
```

Comment:
hmmm I didn't see this function defined in main.cpp?

Reply:
oh good catch, I need to remove that, it's leftover from another attempt to link the click with a callback function !